gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/InstrumentedSpecStore.java

```diff
@@ -164,13 +164,26 @@ public Spec updateSpec(Spec spec) throws IOException, SpecNotFoundException {
     }
   }
 
+  @Override
+  public int getSize() throws IOException {
+    if (!instrumentationEnabled) {
+      return getSizeImpl();
+    } else {
+      long startTimeMillis = System.currentTimeMillis();
+      int size = getSizeImpl();
+      Instrumented.updateTimer(this.getAllTimer, System.currentTimeMillis() - startTimeMillis, TimeUnit.MILLISECONDS);
```

Comment:
looking at something else in this same file, I noticed this (post-merge)...

wouldn't we want a separate `Timer`, to avoid mixing durations from two different operations?  I can include that change in what I'm working on now, if you agree @arjun4084346

Reply:
Ah. will fix in the next PR. thank you for catching

Reply:
cool, no worries.   I added it along with my recent PR already in progress - https://github.com/apache/gobblin/pull/3369